packages/transformer/src/IModelTransformer.ts

```diff
@@ -1132,11 +1146,11 @@ export class IModelTransformer extends IModelExportHandler {
       ? args.provenanceSourceDb
       : args.provenanceDb;
 
+    // FIXME<NICK>: I removed the where e.ecinstanceId not in (0x1, 0xe, 0x10) here. I wonder if we need to special case it for the propagatechangesoption.
```

Comment:
All tests pass with or without the previous WHERE clause. I may have removed this preemptively when I was first trying to support changes to the root elements. 

I'm wondering if in the cases where _options.skipPropagateChangesToRootElements is set to true, 0x1,0xe,0x10 would get filtered out elsewhere in the code anyways.  

I guess thats only true right now because the only way we use forEachTrackedElement in the transformer is to call context.remapElement. Theoretically someone could probably pass a more complex function to forEachTrackedElement that modifies which to me seems like an argument in favor of adding the where clause based off of the truth value of the new skipPropagateChangesToRootElements option. Any thoughts?

Reply:
I would add the where clause. We can always remove it if we find issues.
We shouldn't rely on invariants up in the call tree without asserting them, the order may change eventually.
Hopefully a rally on testing will fix that stuff.

Reply:
not in all cases, depending on the value of the variable I mean